### PR TITLE
Marked Ignite flaky test as MovedToGradle

### PR DESCRIPTION
**Pull request by mihirgune**
The provided ignite flaky test has been moved to Gradle.

Thanks

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2467,7 +2467,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
...
 https://github.com/apache/ignite-3,9641a013665e7012941b2362e9cba64fac4ba0e6,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PersistentPageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,,,
 https://github.com/apache/ignite-3,9641a013665e7012941b2362e9cba64fac4ba0e6,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PersistentPageMemoryMvPartitionStorageTest.uncommittedMultiPageValuesAreReadSuccessfully,ID,,,
-https://github.com/apache/ignite-3,9641a013665e7012941b2362e9cba64fac4ba0e6,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.VolatilePageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,,,
+https://github.com/apache/ignite-3,9641a013665e7012941b2362e9cba64fac4ba0e6,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.VolatilePageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
```
Comment (darko-marinov) [inline pr-data.csv:2470]: Fine to note the commit when Gradle was added, although `pom.xml` was not yet removed. Did you check if the test was `ID` in Gradle; can you update `gr-data.csv` as well?

Comment (mihirgune) [reply pr-data.csv:2470]: Okay. The pom.xml has been removed somewhere in between, would adding the commit at which it was removed to notes be better? I have been trying to check if it is still ID on grade, but haven't been able to run Gradle correctly on this repository.

Comment (darko-marinov) [reply pr-data.csv:2470]: No need to change the commit. I can accept your PR as is now. But you can get more points if you find out how to run NonDex with Gradle. (Hint: it requires changing `build.gradle` file, as the original NonDex for Maven required changing `pom.xml` files so you can run `mvn nondex:nondex ...` before we started running it with `mvn edu.illinois:nondex-maven-plugin:2.1.7:nondex ...`.)

Comment (mihirgune) [reply pr-data.csv:2470]: Yeah, I have added `id('edu.illinois.nondex') version '2.1.7'` as a plugin in my `build.gradle`, and have also added a pluginsManagement block in my `settings.gradle`, but am having some issues. I'm able to import the plugin and run nondex, but it is simply exiting with `BUILD SUCCESS` after executing no tests.

I'll keep looking into it.

Comment (darko-marinov) [reply pr-data.csv:2470]: Did you change `build.gradle` in the subproject? Did you consider using scripts to change the build config?

@hofi1 was our expert for Gradle in particular (and fixing flaky tests in general!) last year. He reported a special case for `ignite-3` (in some older commit at least). The issue was the `ignite-3/buildscripts/java-junit5.gradle` file. This file defines some tasks for the subprojects, which are imported there using the following command:
```
apply from: "$rootDir/buildscripts/java-junit5.gradle"
```
In the `buildscripts/java-junit5.gradle file`, it defines the following block:
```
test {
    useJUnitPlatform()
    ...
```
If you change
```
test {
```
at the start to
```
tasks.withType(Test) {
```
it used to work.

Comment (hofi1) [reply pr-data.csv:2470]: Thanks a lot for mentioning it!

I took a look at it again, and I had to add the following block to the very top of the root gradle file:
```
buildscript {
    repositories {
      maven {
        url = uri('https://plugins.gradle.org/m2/')
      }
    }
    dependencies {
        classpath("edu.illinois:plugin:2.1.1")
    }
}
```

and the following at the subprojects section (root gradle as well):
```
repositories {
            maven {
                url = uri('https://plugins.gradle.org/m2/')
            }
        }
```
then you have to apply the plugin at the end of the subprojects section: 
```
apply plugin: 'edu.illinois.nondex' 
```
 and apply the plugin again at the very bottom of the root gradle file with the following block:
```
apply plugin: 'edu.illinois.nondex'
```

and, as Darko has already meantioned, you have to replace the ``test {`` string in the java-junit5.gradle file in the buildsripts folder with the following string ``tasks.withType(Test) {``

I don't know if this is still the solution to the problem, but last year it worked!

All the best for your progress!

Comment (mihirgune) [reply pr-data.csv:2470]: Thanks for the insights @hofi1 and @darko-marinov. I missed out the part where I need to replace the test string in the java-junit5.gradle file. 
I'll give it a try and let you know!
Cheers!

**Pull request merged**